Sources/AsyncConnection.swift

```diff
@@ -3,7 +3,7 @@ public protocol AsyncConnection: AsyncStream {
 }
 
 extension AsyncConnection {
-    public func open(result: (Void throws -> AsyncConnection) -> Void) throws {
+    public func open(_ result: (Void throws -> AsyncConnection) -> Void) throws {
```

Comment:
That's fair. My only argument is that if we're using the same exact pattern _everywhere_, it makes sense (to me) to create a typealias.